app/views/developers/_details.html.erb

```diff
@@ -0,0 +1,80 @@
+<h2 class="sr-only">Details</h2>
+<div class="space-y-5">
+  <div class="flex items-center space-x-2">
+    <%= render AvailabilityComponent.new(developer: developer, show_unavailable_icon: true) %>
+  </div>
+
+  <% if developer.actively_looking? %>
+    <div class="flex items-center space-x-2">
+      <%= inline_svg_tag "icons/solid/search_circle", class: "h-5 w-5 text-green-500" %>
+      <span class="text-green-700 text-sm font-medium">Actively looking for work</span>
+    </div>
+  <% elsif developer.open? %>
+    <div class="flex items-center space-x-2">
+      <%= inline_svg_tag "icons/solid/search_circle", class: "h-5 w-5 text-gray-400" %>
+      <span class="text-gray-900 text-sm font-medium">Open, but not actively looking for work</span>
+    </div>
+  <% elsif developer.not_interested? %>
+    <div class="flex items-center space-x-2">
+      <%= inline_svg_tag "icons/solid/x_circle", class: "h-5 w-5 text-gray-400" %>
+      <span class="text-gray-900 text-sm font-medium">Not interested in work right now</span>
+    </div>
+  <% end %>
+
+  <% if developer.role_type? %>
+    <div class="mt-6 border-t border-gray-200 pt-6 space-y-8">
+      <div>
+        <h2 class="text-sm font-medium text-gray-500">Interested in roles</h2>
+        <ul role="list" class="mt-3 space-y-3">
+          <% Developer.role_types.each do |role| %>
+            <% if developer.send("#{role}?") %>
+              <li class="flex items-center space-x-2">
+                <%= inline_svg_tag "icons/solid/check_circle", class: "h-5 w-5 text-gray-400" %>
+                <span class="text-gray-900 text-sm font-medium"><%= role.to_s.humanize %></span>
+              </li>
+            <% end %>
+          <% end %>
+        </ul>
+      </div>
+    </div>
+  <% end %>
+
+  <div class="mt-6 border-t border-gray-200 py-6 space-y-8">
+    <div>
+      <ul role="list" class="space-y-3">
+        <% if developer.website.present? %>
+          <li>
+            <%= render ExternalLinkComponent.new(developer.website) %>
+          </li>
+        <% end %>
+
+        <% if developer.github.present? %>
```

Comment:
These could also probably be extracted. Params being:

* `developer`
* `attribute`
* `icon`
* `url_prefix`